runbotics/runbotics-scheduler/src/scheduler-database/authority-feature-key/authority-feature-key.entity.ts

```diff
@@ -0,0 +1,22 @@
+import { Entity, JoinColumn, JoinTable, ManyToMany, ManyToOne, PrimaryColumn } from 'typeorm';
+import { FeatureKey } from '../feature-key/feature-key.entity';
+import { Role as RoleEnum } from 'runbotics-common';
+import { Authority } from '../authority/authority.entity';
+import { FeatureKey as FeatureKeyEnum } from 'runbotics-common';
+
+@Entity({ name: 'authority_feature_key' })
+export class AuthorityFeatureKey {
+    @ManyToOne(() => Authority, { eager: true })
+    @JoinColumn({ name: 'authority', referencedColumnName: 'name' })
+    authorities: Authority[];
```

Comment:
Should we add here cascade? 'onDelete: 'CASCADE' What do you think?

Reply:
Yeah, only authority+feature-key pair makes sense in this table, on the other hand we delete authorities hardly ever and there' no endpoint for modification of authorites. So I wouldn't add onDelete cascade, because there's more benefits from it being security feature (in case someone removes authority by mistake, thinking it's unsed) than maintenance errand (when we decide to really remove some authority, which happens hardly ever and is just one additional query)